aptos-move/framework/aptos-stdlib/sources/math128.move

```diff
@@ -29,8 +28,9 @@ module aptos_std::math128 {
         }
     }
 
-    /// Returns a * b / c going through u128 to prevent intermediate overflow
+    /// Returns a * b / c going through u256 to prevent intermediate overflow
     public inline fun mul_div(a: u128, b: u128, c: u128): u128 {
+        assert!(c != 0, std::error::invalid_argument(4));
```

Comment:
what's 4 here?

Reply:
I thought the best practice is to make 4 a constant of this file?

Reply:
I guess I should fix the comment, that I thought I had.

You cannot have a constant, because it doesn't copy between inlines.  So, the best we can do is to have a common number, in this case 4 is used for fixed_point32 when a divide by 0 happens